tests/test_iqm_provider.py

```diff
@@ -541,7 +544,7 @@ def test_create_run_request(backend, circuit, create_run_request_default_kwargs,
     when(backend.client).submit_run_request(run_request).thenReturn(uuid.uuid4())
 
     assert backend.create_run_request(circuit_transpiled, **options) == run_request
```

Comment:
Can remove `options` also from here (since it just has transpilation options), I forgot it after the last change in my PR.

```suggestion
    assert backend.create_run_request(circuit_transpiled) == run_request
```